components/CostsCard.vue

```diff
@@ -61,6 +60,9 @@
           @mouseover="hideTooltips = false"
         />
       </div>
+      <p class="text-end fw-lighter vsl-display">
+        Average value of life: {{ formatCurrency(appStore.currentScenario.result.data?.average_vsl || 0) }} USD
+      </p>
```

Comment:
I'm no longer at all sure what the VSL value really represents. @pratikunterwegs , can you review the wording here? We might need to make it clearer that this is something variable / country-dependent, so maybe "National value of life used".

Also, I think I recall that the unit is not USD, but international dollars, but I'm not sure.

Reply:
I think this is sufficient, and "Value of a statistical life" which is the expansion of VSL would also be appropriate, and perhaps preferable as it suggests that it's a broad generalisation.

It's explicitly not the national value-of-life, but a measure calculated using the country-specific GNI. It generalises well - otherwise, we'd have to implement the specific formula each country uses for its national calculations, including for countries where this isn't strictly defined or readily available.

The unit is indeed international dollars.

Reply:
On international dollars: https://datahelpdesk.worldbank.org/knowledgebase/articles/114944-what-is-an-international-dollar

Reply:
```suggestion
      <p class="text-end fw-lighter vsl-display">
        Value of a statistical life: {{ formatCurrency(appStore.currentScenario.result.data?.average_vsl || 0) }} Int$
      </p>
```